PreciseAlloy.Frontend/xpack/root/RootNav.tsx

```diff
@@ -30,16 +30,28 @@ export default function RootNav({ routes: routesProp }: Props) {
     debounce((text: string) => {
       const tempRoutes: RootItemModel[] = [],
         tempRoutesSearch: RootItemModel[] = [];
+      const conditions: (item: RootItemModel) => boolean = (item: RootItemModel) => {
```

Comment:
The name of the function should make sense. We should rename it like hasPageNode

Reply:
I agree with Hung. "conditions" is a recursive function check if item name or its children's name included search term. Function name could be "isItemIncludedText" or shortly "includedText".